reference/conanfile/tools/microsoft.rst

```diff
@@ -208,6 +208,23 @@ The MSBuildToolchain files can configure:
 One of the advantages of using toolchains is that they can help to achieve the exact same build
 with local development flows, than when the package is created in the cache.
 
+conf
+++++
+
+``MSBuildToolchain`` is affected by these :ref:`[conf]<global_conf>` variables:
+
+- ``tools.microsoft.msbuildtoolchain:compile_options`` dict-like object of extra compile options to be added to ``<ClCompile>`` section.
+  The dict will be translated as follow: ``<[KEY]>[VALUE]</[KEY]>``.
+- ``tools.build:cxxflags`` list of extra C++ flags that will be appended to ``<AdditionalOptions>`` section from ``<ClCompile>`` and ``<ResourceCompile>`` one.
+- ``tools.build:cflags`` list of extra of pure C flags that will be appended to ``<AdditionalOptions>`` section from ``<ClCompile>`` and ``<ResourceCompile>`` one.
+- ``tools.build:sharedlinkflags`` list of extra linker flags that will be appended to ``<AdditionalOptions>`` section from ``<Link>`` one.
+- ``tools.build:exelinkflags`` list of extra linker flags that will be appended to ``<AdditionalOptions>`` section from ``<Link>`` one.
+- ``tools.build:defines`` list of preprocessor definitions that will be appended to ``<PreprocessorDefinitions>`` section from ``<ResourceCompile>`` one.
+
+.. note::
+
+    All the "sections" mentioned above can be found out in *conantoolchain_<config>.props* file created by :command:`conan install` command.
+
```

Comment:
I think we can omit this note, it does not seem like a relevant information for the users.

Reply:
Yeah, I wanted to know your opinion about it because I got doubts about adding it or not. Agree 😉